.github/workflows/release.yml

```diff
@@ -50,16 +54,21 @@ jobs:
         cp projects/cm_mcu/gcc/cm_mcu.axf cm_mcu_noecn001_rev2.axf
         make clean REV2=1
         make -k REV2=1
-        cp projects/cm_mcu/gcc/cm_mcu.bin cm_mcu_rev2.bin
-        cp projects/cm_mcu/gcc/cm_mcu.axf cm_mcu_rev2.axf
-        cp projects/boot_loader/gcc/bl_main.bin bl_main_rev2.bin
-        cp projects/boot_loader/gcc/bl_main.axf bl_main_rev2.axf
+        mkdir REV2
+        cp projects/cm_mcu/gcc/cm_mcu.bin REV2/cm_mcu_rev2.bin
+        cp projects/cm_mcu/gcc/cm_mcu.axf REV2/cm_mcu_rev2.axf
+        cp projects/boot_loader/gcc/bl_main.bin REV2/bl_main_rev2.bin
+        cp projects/boot_loader/gcc/bl_main.axf REV2/bl_main_rev2.axf
+        make release
+        mv PL_MEM_CM_rev2.xml REV2
+        tar zcvpf cm_mcu_rev2.tar.gz REV2
      - name: test
        run: pwd; ls -R ; find . -type f -name '*.axf' -print
      - uses: "marvinpinto/action-automatic-releases@latest"
        with:
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
           prerelease: false
           files: |
-            ./cm_mcu*.*
-            ./bl_main*.*
+            ./REV1/*
+            ./REV2/*
+            *.tar.gz
```

Comment:
It looks reasonable to me that we will keep tarfiles in `cm_mcu/REV*/` folders. Note that this can only be tested once merge to master is done.